ONSdigital/ssdc-rm-support-tool
# check pack code is unique in ui and server side:

# Motivation and Context
Stop over writing existing rows for the same packCode for print & SMS templates

# What has changed
Checks in UI and server side

# How to test?
Get it running.  Create an SMSTemplate, then try to recreate with the same packCode.
Repeat for PrintTemplate.
I had to comment out ui checking code to check that the server side also does this check as well.  Couldn't think of a nicer way of doing it.

# Links
https://trello.com/c/KxFo10XC/2962-trying-to-create-a-print-sms-template-with-existing-name-replaces-the-old-one

# Screenshots (if appropriate):
